tox.ini

```diff
@@ -11,10 +10,28 @@ deps =
 commands = python setup.py test
 
 [testenv:coverage]
+allowlist_externals = coverage
 basepython = python3.8
 deps =
     {[testenv]deps}
-    coveralls
+    ; coveralls
 commands =
     coverage run --source=tapioca setup.py test
-    coveralls
+    ; coveralls
```

Comment:
Commented code here. Do you need to activate coveralls? I'll make a repo admin now, in case that's helpful.

Reply:
In fact, you're an admin already

Reply:
I was testing without coveralls, It will need some extra configuration yet